src/movement.cpp

```diff
@@ -405,12 +406,12 @@ uint32_t MoveEvents::onCreatureMove(Creature* creature, const Tile* tile, MoveEv
 	}
 
 	for (size_t i = tile->getFirstIndex(), j = tile->getLastIndex(); i < j; ++i) {
-		Thing* thing = tile->getThing(i);
+		auto thing = tile->getThing(i);
```

Comment:
Could this be `const auto`
```suggestion
		const auto thing = tile->getThing(i);
```